gapic-generator/lib/google/gapic/path_template/parser.rb

```diff
@@ -14,57 +14,60 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-require "google/gapic/template/segment"
+require "google/gapic/path_template/segment"
 
 module Google
   module Gapic
-    module Template
-      # A URI template parser.
+    module PathTemplate
+      # A URI path template parser.
       #
-      # @!attribute [r] template
-      #   @return [String] The template to be parsed.
+      # @see https://tools.ietf.org/html/rfc6570 URI Template
+      #
+      # @!attribute [r] path_template
+      #   @return [String] The URI path template to be parsed.
       # @!attribute [r] segments
-      #   @return [Array<Segment|String>] The segments of the parsed template.
+      #   @return [Array<Segment|String>] The segments of the parsed URI path
+      #     template.
       class Parser
         # @private
-        # /((?<positional>\*\*?)|{(?<name>[^\/]+?)(?:=(?<template>.+?))?})/
+        # /((?<positional>\*\*?)|{(?<name>[^\/]+?)(?:=(?<path_template>.+?))?})/
         TEMPLATE = %r{
```

Comment:
I would rename the constant instead.

```suggestion
        PATH_TEMPLATE = %r{
```